### PR TITLE
UI cleanups and fixes

### LINKED ISSUE #1465: Drop the backticks from the show-controller command example on the login page
Because if you did, I could double-click that line, copy and paste directly into the terminal.

With the backticks, it's more like:
```
$ `juju show-controller --show-passwords`
local.dev:: command not found
```

I have to carefully select that text inside the ticks, or type it all out. I'm too lazy for all that.

### LINKED ISSUE #1469: Add the environment name to the page title
I suggest something like "prod - Juju GUI". This will make it easier to see which tab is which environment and by putting the name first the user can still see the name if they have many tabs open.

### LINKED ISSUE #1481: Login screen always says 'Sandbox'
When logging into the GUI no matter what the model name is it always says "Sandbox"

screenshot from 2016-03-18 15 42 20

### LINKED ISSUE #1498: In the inspector unit listings for errors, pending and uncommitted change the generic "Select all units" to "Select pending units", "Select uncommitted units".

### PR DESCRIPTION
**Pull request by huwshimi**
In this branch:
- disabled the commit button on the deploy summary if there are no changes
- removed the quotes around the password command on the login screen (fixes #1465)
- add the model name to the page title (fixes #1469)
- correctly display the model name on the login screen (fixes #1481)
- change the titles for the select all actions on the unit lists (fixes #1498).

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4764/
Test PASSed.

**Comment by huwshimi** [comment]
QA

Disabled the commit button on the deploy summary if there are no changes:
- visit `/deploy/summary/:flags:/blues/` with no changes to your model
- the Deploy button should be disabled
- close the summary and make a change
- open the deploy summary again and the button should be enabled

Change the titles for the select all actions on the unit lists (fixes #1498):
- add a service
- click on the unit list for uncommitted units
- check that the select all option reads "Select uncommitted units"
- turn on the simulator and check that the pending units option reads "Select pending units"

Removed the quotes around the password command on the login screen (fixes #1465):
- spin up a real model
- check that the command underneath the login for for how to find your password does not have ` marks around it

Correctly display the model name on the login screen (fixes #1481):
- on the login screen check that the model name is correct above the form

Add the model name to the page title (fixes #1469):
- in the real model check that your browser displays the model name in the tab/window title

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -686,20 +686,20 @@ YUI.add('juju-gui', function(Y) {
...
             Find your password with<br />
-            <code>`juju api-info --password password`</code>
+            <code>juju api-info --password password</code>
           </p>);
       }
       document.getElementById('loading-message').style.display = 'none';
       ReactDOM.render(
         <window.juju.components.Login
-          envName={this.db.environment.get('name') || 'Sandbox'}
+          envName={this.get('jujuEnvUUID') || this.db.environment.get('name')}
```
Comment (hatched) [inline jujugui/static/gui/src/app/app.js]: I'm a little concerned that this might actually get a 'uuid' instead of model name. Something to check in qa.

**Comment by hatched** [comment]
:+1:

**Comment by bac** [comment]
QA-hmmmm

There are five QA sections.  All but the second were great.

For the second I did not get "Select uncommitted units". I only saw "Select all units". I'm not sure why.  This was when running via `make run` and when deployed via the charm.

**Comment by bac** [comment]
Code :+1:

File: jujugui/static/gui/src/app/app.js (hunk 1 above)
Comment (huwshimi) [inline jujugui/static/gui/src/app/app.js]: I just copied this from how we get the name for the breadcrumb.

**Comment by hatched** [comment]
This QA'd properly for me so I'm going to assume that your issue @bac was a caching one. If it creeps back up I guess we'll know where to look :)

Thanks @huwshimi 

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**